.github/workflows/tests.yml

```diff
@@ -66,19 +66,19 @@ jobs:
     name: Go tests
     env:
       USE_EXISTING_CLUSTER: true
-      CORE_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/harbor-core:v2.2.1
-      EXPORTER_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/harbor-exporter:v2.2.1
-      JOBSERVICE_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/harbor-jobservice:v2.2.1
-      PORTAL_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/harbor-portal:v2.2.1
-      REGISTRYCTL_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/harbor-registryctl:v2.2.1
-      REGISTRY_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/registry-photon:v2.2.1
-      CHARTMUSEUM_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/chartmuseum-photon:v2.2.1
-      NOTARYSERVER_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/notary-server-photon:v2.2.1
-      NOTARYSIGNER_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/notary-signer-photon:v2.2.1
-      TRIVY_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/trivy-adapter-photon:v2.2.1
-      HARBORCLUSTER_MINIO_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/minio:2021-04-06T23-11-00Z
-      HARBORCLUSTER_REDIS_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/redis:5.0-alpine
-      HARBORCLUSTER_POSTGRESQL_DOCKER_IMAGE_2_2_1: ghcr.io/goharbor/spilo-12:1.6-p3
+      CORE_DOCKER_IMAGE_2_3_0: ghcr.io/goharbor/harbor-core:v2.3.0
```

Comment:
It seems we should provide repository and tag suffix ENVs to simplify the image path settings here.

Reply:
Issue created https://github.com/goharbor/harbor-operator/issues/701 and will be fixed in another PR.